tests/test_config_flow_password_selector.py

```
@@ -0,0 +1,87 @@
+
+class PasswordSelectorTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.tree = ast.parse(CONFIG_FLOW.read_text(encoding="utf-8"))
+
+    def _password_schema_values(self) -> list[ast.expr]:
+        """Every value bound to a ``vol.Required("password")`` schema key."""
+        values: list[ast.expr] = []
+        for node in ast.walk(self.tree):
+            if not isinstance(node, ast.Dict):
+                continue
+            for key, value in zip(node.keys, node.values):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)